src/freetype/ft-ogl.ads

```diff
@@ -0,0 +1,44 @@
+--------------------------------------------------------------------------------
+-- Copyright (c) 2012, Felix Krause <flyx@isobeef.org>
+--
+-- Permission to use, copy, modify, and/or distribute this software for any
+-- purpose with or without fee is hereby granted, provided that the above
+-- copyright notice and this permission notice appear in all copies.
+--
+-- THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+-- WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+-- MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+-- ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+-- WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+-- ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+-- OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+--------------------------------------------------------------------------------
+
+with GL.Objects.Programs;
+with GL.Types;
+with GL.Types.Colors;
+with GL.Uniforms;
+
+package FT.OGL is
+   type Character_Record is private;
```

Comment:
There is no way of getting access to a `Character_Record` value with the public subprograms in this package, so there is no need to expose this declaration.

Reply:
Fixed.